README.rst

```diff
@@ -94,6 +94,15 @@ or
 
 $ ./host_os.py build-package --help
 
+* Build custom software
+
+The build code keeps a cache of all source code management (Git, Mercurial, SVN)
+repositories so that they do not need to be downloaded on every build. If it is
+desired to change the source code of a package, you may edit the package local
+repository content. To build the package from the custom content, either use a
+file:// URL pointing to the local repository in the package metadata (YAML) 
```

Comment:
"it" would be confusing in this context IMHO. A reader would not know if it means "package", "custom content" or "local repository"